utils/coreutils/coreconsts.go

```diff
@@ -49,6 +49,9 @@ const (
 	CI                      = "CI"
 	ServerID                = "JFROG_CLI_SERVER_ID"
 	TransitiveDownload      = "JFROG_CLI_TRANSITIVE_DOWNLOAD"
+	// Envs used to determine command types for usage reports
```

Comment:
Let's add more information to this comment, explaining that those variables can be set by the `setup-jfrog-cli` GitHub Action, and also provide some background about the use case.
This will help future readers of this code understand the context.